submit.yml.erb

```diff
@@ -85,7 +87,7 @@ script:
         KUBECONFIG: "/dev/null"
       port: "8080"
       cpu: "<%= num_cores %>"
-      memory: "<%= num_cores.to_i * 4 %>Gi"
+      memory: "<%= memory_mb %>Mi"
```

Comment:
Can you leave a comment here about doing the memory calculation because of hyperthreading.

Reply:
Comment added where the ratio is determined, can move if needed.